ios/FluentUI.Tests/DatePickerControllerTests.swift

```diff
@@ -13,13 +13,13 @@ class DatePickerControllerTests: XCTestCase {
     let endDate = Date(timeIntervalSince1970: DatePickerControllerTests.testDateInterval).adding(days: 1)
 
     func testDateRangeInit() {
-        let datePicker = DatePickerController(startDate: startDate, endDate: endDate, mode: .dateRange, rangePresentation: .paged, titles: nil)
+        let datePicker = DatePickerController(startDate: startDate, endDate: endDate, mode: .dateRange, rangePresentation: .paged, titles: nil, leftBarButtonItem: nil, rightBarButtonItem: nil)
```

Comment:
if the datepickercontroller function has = nil in the optional parm we won't need these changes right?

Reply:
@harrieshin, the initializer of DatePickerController requires all parameters, that can be null
I have followed the identical pattern used for Titles (the last nullable parameter before my changes)

Reply:
right so if you change the initalizer to have the default param then you don't need these changes

Reply:
Done it and removed the nil parameters from the tests.
For coherence, I have made the last two params optional both in DatePickerController and DateTimePickerController, even if the last one does not have tests.